go.sum

```diff
@@ -869,6 +869,8 @@ github.com/qri-io/compare v0.1.0 h1:A/MRx3uEnJ/iMjfJY1VOqH9CYs9zFSEYaFVeXuGfmis=
 github.com/qri-io/compare v0.1.0/go.mod h1:i/tVuDGRXVxhuZ8ZUieF23u6rQ6wLGJl7KKWpoMRaTE=
 github.com/qri-io/dag v0.2.1-0.20191025201336-254aa177fbd7 h1:hujyzdufAz3Hi6I0I0yKyJTLCM9eoLamKhlpwaqE7ec=
```

Comment:
mind running `go mod tidy` and amending this commit? It'll drop a bunch of stale imports from `go.sum` like this one